AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -3689,6 +3688,7 @@ bmwfaq.org#?#.Notices[-ext-has='a[href="//www.bmwfaq.org/help/cookies"]']
 ! JS rules
 !
 freeyou.ag#%#//scriptlet('remove-class', 'fixed-body', 'body,html')
+rockpapershotgun.com#%#AG_onLoad(function(){var a=new MutationObserver(function(){if(document.querySelector(".optanon-alert-box-wrapper")){document.querySelector("button.accept-cookies-button").click(); a.disconnect();}});a.observe(document,{childList: !0, subtree: !0});setTimeout(function(){a.disconnect()},5000)});
```

Comment:
Is something unavailable if popup is just hidden?

Reply:
Yes, videos for example
https://www.rockpapershotgun.com/videos